package.json

```diff
@@ -43,7 +43,6 @@
     "filesize": "^6.1.0",
     "gzip-size": "^5.1.1",
     "lodash": "^4.17.20",
-    "mkdirp": "^1.0.4",
```

Comment:
Removing this line is not enough - `package-lock.json` should also be updated. Run `npm rm mkdirp` to properly remove dependency please.